templates/prelogin/demo.html

```diff
@@ -0,0 +1,126 @@
+{% extends 'prelogin/base.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block stylesheets %}
+    <link href="{% static "app_manager/js/vellum/style.css" %}" type="text/css" rel="stylesheet"/>
+    <style>
+        .form-signup {
+            margin-left: 0;
+        }
+    </style>
+{% endblock stylesheets %}
+
+{% block js-inline %}
+    <script>
+        $(function(){
+            var signUpFormHandler = function() {
+                var submittedForm = $(this);
+                submittedForm.find(".btn[type='submit']").attr('disabled', 'disabled').prepend('<i class="fa fa-refresh fa-spin icon-refresh icon-spin"></i>');
+                kmqPushSafe(['record', 'Clicked "Sign up" from demo', {}, function() {
+                    submittedForm.off('submit', signUpFormHandler).submit();
+                }]);
+                return false;
+            };
+            $('form.form-signup').on('submit', signUpFormHandler);
```

Comment:
@amsagoff yes, figuring out a test plan sounds good, I'll set up some time for us to talk

@benrudolph yep, I stole this from disable-on-submit & should just use the actual disable-on-submit code, will update